### PR TITLE
Updated alt attributes of some images in "getting_started_with_web" guide.

### PR DESCRIPTION
**Pull request by ghost**
#### Summary

There are some missing or empty alt attributes to images in "getting_started_with_web" guide in learn section. So I have updated those with some good small descriptions explaining about images.
#### Supporting details

List of missing alt attributes: https://docs.google.com/spreadsheets/d/1cDHee6mwDMlY6zANEmg8MjR8tL-wrCS86uawpWZ4XbA/edit?usp=sharing

#### Related issues

This pull request related to issue https://github.com/mdn/content/issues/19334
and add some fixes to it.
#### Metadata

- [x] Updates alt attributes of some images in "Getting started with web" guide.
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by iSatVeerSingh** [comment]
Note :- I did a typo in Title of this PR.

There should be "getting_started_with_web"

File: files/en-us/learn/getting_started_with_the_web/javascript_basics/index.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ However, getting comfortable with JavaScript is more challenging than getting co
     myHeading.textContent = 'Hello world!';
     ```
 
-5. Make sure the HTML and JavaScript files are saved. Then load `index.html` in your browser. You should see something like this:![](hello-world.png)
+5. Make sure the HTML and JavaScript files are saved. Then load `index.html` in your browser. You should see something like this:![New H1 heading element created and rendered in DOM using javascript](hello-world.png)
```
Comment (estelle) [inline files/en-us/learn/getting_started_with_the_web/javascript_basics/index.md:58]:
```suggestion
5. Make sure the HTML and JavaScript files are saved. Then load `index.html` in your browser. You should see something like this:!

[Heading "hello world" above a firefox logo](hello-world.png)
```

File: files/en-us/learn/getting_started_with_the_web/javascript_basics/index.md [hunk 2]
```diff
@@ -218,7 +218,7 @@ An `{{Glossary("operator")}}` is a mathematical symbol that produces a result ba
       <td><code>-</code>, <code>*</code>, <code>/</code></td>
       <td>
         <code
-          >9 - 3;<br />8 * 2; // multiply in JS is an asterisk<br />9 / 3;</code
+>9 - 3;<br />8* 2; // multiply in JS is an asterisk<br />9 / 3;</code
```
Comment (estelle) [inline files/en-us/learn/getting_started_with_the_web/javascript_basics/index.md:221]:
```suggestion
>9 - 3;<br />8 * 2; // multiply in JS is an asterisk<br />9 / 3;</code
```

File: files/en-us/learn/getting_started_with_the_web/javascript_basics/index.md [hunk 3]
```diff
@@ -479,7 +479,7 @@ In human language, this means: If `myName` has no value, run `setUserName()` aga
 
 If you have followed all the instructions in this article, you should end up with a page that looks something like the image below. You can also [view our version](https://mdn.github.io/beginner-html-site-scripted/).
 
-![](website-screen-scripted.png)
+![Final look of HTML page after creating elements and adding DOM Manipulation (user interactions) using JavaScript](website-screen-scripted.png)
```
Comment (estelle) [inline files/en-us/learn/getting_started_with_the_web/javascript_basics/index.md:482]:
```suggestion
![Final look of HTML page after creating elements: a header, large centered logo, content, and a button](website-screen-scripted.png)
```

File: files/en-us/learn/getting_started_with_the_web/what_will_your_website_look_like/index.md [hunk 4]
```diff
@@ -52,7 +52,7 @@ You should still have your paragraphs and title from earlier. Keep these close b
 
 To choose a color, go to [the Color Picker](/en-US/docs/Web/CSS/CSS_Colors/Color_picker_tool) and find a color you like. When you click on a color, you'll see a strange six-character code like `#660066`. That's called a _hex code_ (short for hexadecimal), and represents your color. Copy the code down somewhere safe for now.
 
-![](color-picker.png)
+![Screenshot of Color-Picker-Tool on MDN Docs website](color-picker.png)
```
Comment (estelle) [inline files/en-us/learn/getting_started_with_the_web/what_will_your_website_look_like/index.md:55]:
```suggestion
![Color-Picker-Tool on MDN Docs website with RGB, HSL, and HEX colors ](color-picker.png)
```

**Review by estelle — APPROVED** [review]
Looks good. I made a few edits. Thank you!

**Pull request merged**